tests/acceptance/test_acceptance/test_decide.py

```diff
@@ -78,6 +110,53 @@ def test_decide__feature(session_obj, flag_key, expected_response, expected_stat
     resp.raise_for_status()
 
 
+@pytest.mark.parametrize(
+    "flag_key, expected_response, expected_status_code, forced_flag, forced_rule, forced_variation", [
+        ("feature_2", expected_forced_decision_without_rule_key, 200, "feature_2", None, "variation_1"),
+        ("feature_2", expected_forced_decision_with_rule_key, 200, "feature_2", "feature_2_test", "variation_2")
+    ],
+    ids=["variation_1", "16931381940"])
+def test_decide_with_forced_decision__feature(session_obj, flag_key, expected_response, expected_status_code, forced_flag, forced_rule, forced_variation):
```

Comment:
What about other forced decision api's? @jaeopt what do you think?

Reply:
Also, could we have cariation override for forced decisions in /v1/override endpoint? Like here: https://github.com/optimizely/agent/blob/yasir/forced-decision-acceptance/tests/acceptance/test_acceptance/test_overrides.py